### ISSUE TITLE
Multiple megamenus

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#36]
Hi,

When using Orange Megamenu with multiple instances, opening a second megamenu does not hide the first opened megamenu. Quite annoying. 

Could you please help with this, at least providing a simple way to close a megamenu?

Using this I do know if there's already an opened megamenu: 
```
jQuery(document).ready(function ($) {
    $('.nav-link[data-toggle="collapse"]').on('click', function(e) {
        $open = $('.mega-menu > [aria-expanded="true"]');        
    });
});
```
But that may not be the better approach. Thanks :)

**Comment by ygatesoupe** [comment]
Hi,

Fixed in commit 8760940e265b113ce63e4b0b7debb1a49c74a9bb

**Comment by ffoodd** [comment]
Hi,

for now as I'm sticking to alpha 5 on the project I'm working on, I didn't try to incorporate your fix in Boosted sources. I preferred a much more simple way:

```
jQuery(document).ready(function ($) {
    $('.nav-link[data-toggle="collapse"]').on('click', function(e) {
        $open = $('.mega-menu > [aria-expanded="true"]');

        if($open.length) {
            $open.collapse('hide');
        }
    });
});
```

This may be useful for other projects sticking to alpha 5. But your fix did the trick so I'm closing the issue, thanks :)
